corehq/apps/export/static/export/js/models.js

```diff
@@ -214,7 +237,17 @@ hqDefine('export/js/models.js', function () {
     UserDefinedTableConfiguration.prototype = Object.create(TableConfiguration.prototype);
 
     UserDefinedTableConfiguration.prototype.customPathToNodes = function() {
+        var rowColumn,
+            nestedRepeatCount;
         this.path(utils.customPathToNodes(this.customPathString()));
+
+        // Update the rowColumn's repeat count by counting the number of
+        // repeats in the table path
+        nestedRepeatCount = _.filter(this.path(), function(node) { return node.is_repeat() }).length;
```

Comment:
super minor, but this could go inside the if block so it doesn't get run unless it's going to be used